Pipfile.lock

```diff
@@ -585,11 +585,11 @@
         },
         "identify": {
             "hashes": [
-                "sha256:de7129142a5c86d75a52b96f394d94d96d497881d2aaf8eafe320cdbe8ac4bcc",
-                "sha256:e0dae57c0397629ce13c289f6ddde0204edf518f557bfdb1e56474aa143e77c3"
+                "sha256:2179e7359471ab55729f201b3fdf7dc2778e221f868410fedcb0987b791ba552",
+                "sha256:2a5fdf2f5319cc357eda2550bea713a404392495961022cf2462624ce62f0f46"
             ],
-            "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
-            "version": "==1.5.14"
+            "markers": "python_full_version >= '3.6.1'",
```

Comment:
Can you reset this to `upstream/master`.

Reply:
I didn't know that happened, sure

Reply:
fixed as of https://github.com/python-discord/sir-lancebot/pull/609/commits/825080e805787afc1f36a80b4fa3f73040ebdc75

Reply:
Quoting https://github.com/Kronifer/sir-lancebot/blob/earth_photos/Pipfile.lock#L591

Reply:
I am, and it is still not solved.

Reply:
Oh wow, Im stupid! Sorry @Shivansh-007 , I looked at it wrong

Reply:
Hi @Shivansh-007 , I cant do this as it breaks pipenv and the packages, sorry!

Reply:
It doesn’t beak them. You are doing `pipenv lock`, am asking you to reset it to what it is at  the main branch master that is, what it is [here](https://github.com/python-discord/sir-lancebot/blob/master/Pipfile.lock)

Reply:
sorry, im not sure how to do this, what would i use?

Reply:
Copy the `Pipfile.lock` from python-discord/sir-lancebot and paste it into yours.

Reply:
ok

Reply:
@Shivansh-007 I did this but as soon as I ran pipenv sync --dev locally it reset this

Reply:
it failed identify install when I made the change